test_data/third_party/2and3/six_test.py

```diff
@@ -0,0 +1,17 @@
+import pip  # type: ignore
+
+pip.main('install six==1.10.0'.split())
```

Comment:
I've extracted the code for installing requirements into a pytest fixture that automatically installs *_requirements.txt before running tests from *_test.py. You can run a single test separately and pytest will ensure requirements are installed.